USERS.md

```diff
@@ -7,26 +7,27 @@ TEMPLATE:
 
 -->
 
-- https://arubacao.com **Christopher Lass** (Software Engineer and DevOps)
+- https://arubacao.com - **Christopher Lass** (Software Engineer and DevOps)
 - https://blog.agate.pw/ - **Marco Agate** (System Engineer, DevOps)
 - https://blog.lepape.me/ - **François Le Pape** (Student & developer freelance)
 - https://etra0.github.io/ - **Sebastián Aedo** (Computer Science student, interested in Game Hacking)
-- https://fazi1058.github.io **Faezeh Roeinfard** (Student)
+- https://fazi1058.github.io - **Faezeh Roeinfard** (Student)
 - https://felixleger.com/ - **Félix Léger** (Sofware Developer and DevOps)
 - https://guilhermesteves.dev/ - **Guilherme Esteves** (Software Engineer, Writer, Speaker)
 - https://hesec.de - **Patrick Hener** (Security Researcher, Pentester and Coder)
 - https://imsalone.com/ - **Iancu makes stuff alone** (Indie Game Dev)
 - https://jonathan.rico.live/ - **Jonathan Rico** (Electronics Engineer)
 - https://kartiniteknologi.id/ - **Kartini Teknologi team** (Tech podcast)
-- https://tug.ro/ — **Ashish Ranjan** (Software Engineer)
-- https://protocod.gitlab.io/blog/ **protocod** (Web Developer)
+- https://tug.ro/ - **Ashish Ranjan** (Software Engineer)
+- https://protocod.gitlab.io/blog/ - **protocod** (Web Developer)
 - https://shinytoyrobots.com - **shinytoyrobots** (Eclectic journaling and essays)
 - https://simeononsecurity.ch/ - **SimeonOnSecurity** (Security and Automation Blog)
 - https://www.miroslavbucek.cz - **Miroslav Buček** (Product Manager, Smart Home company owner)
 - https://fmg3d.com/ - **Finn M Glas** (Aspiring Physicist, Software Developer)
 - https://nali.org/ - **Nick Ali** (Marketer)
 - https://abhinav031.netlify.app/ - **Abhinav Sriram** (Digital Artist, Hobbyist)
-- https://codigonovato.netlify.app **Javier Rangel** (Dev and notes)
+- https://codigonovato.netlify.app - **Javier Rangel** (Dev and notes)
 - https://easyitblog.info/ - **Ivan Vaskevych** (Staff engineer)
-- https://0days.io **Frederik Bøgeskov Krogsgaard** (Cyber & Information Security)
-- https://codejuggle.dj/ **cornelk** (Software Engineer and DevOps)
+- https://0days.io - **Frederik Bøgeskov Krogsgaard** (Cyber & Information Security)
+- https://codejuggle.dj/ - **cornelk** (Software Engineer and DevOps)
+- https://walmyrcarvalho.com.br/ **Walmyr Carvalho** (Senior Android Engineer and Google Developer Expert)
```

Comment:
Ha! It's funny that you fixed all the missing dashes and forgot to put it by yourself 😅

Reply:
HAHAH wow, so lame, let me fix that!

Reply:
No problem 🙈